docs/snippets/pre_commit.rst

```diff
@@ -0,0 +1,11 @@
+**pydocstyle** can be included as a hook for `pre-commit`_.  The easiest way to get
+started is to add this configuration to your ``.pre-commit-config.yaml``:
+
+.. code-block:: yaml
+
+    -   repo: https://github.com/pycqa/pydocstyle
+        rev: 3.0.0  # pick a git hash / tag to point to
```

Comment:
so there's a few potential problems with this, I would have left it how you had before 🤷‍♂️ 

1. `3.0.0` is a tag that doesn't exist in this repo
2. if you pick one that does exist (`2.0.0` for example) it will either be out of date when a new release happens, or will need to be updated as the project releases.

Reply:
Good point.
We can make it dynamic, I came up with 2 options
1. We can update bump version to cover the `rst` file

2. We can use `parsed-literal`
```suggestion
.. parsed-literal::

    -   repo: https://github.com/pycqa/pydocstyle
        rev: \ |version| \  # pick a git hash / tag to point to
        hooks:
        -   id: pydocstyle
```
However it won't get rendered with highlighting as usual code block.
![image](https://user-images.githubusercontent.com/7480694/49342498-711dc280-f65c-11e8-8661-c16492b529d7.png)

Reply:
in case with problems with styling, there is a suggested workaround: 
`https://github.com/sphinx-doc/sphinx/issues/3306`

Reply:
@Nurdok Could you please let me know if you like the solution?

Reply:
@1oglop1 thanks for pinging me, sorry.

Using `|version|` seems fine to me.

Reply:
@Nurdok Thanks for prompt response, please review the update.

Reply:
@1oglop1 LGTM, but repeating comment on the main thread: can you please point to a rendered version of the docs? e.g., create a readthedocs project for your fork or something like that.

I just want to take a look to see the rendering here look okay in practice, and I'll merge.